APIKit/APIKit.swift

```diff
@@ -146,13 +111,17 @@ public class API: NSObject, NSURLSessionDelegate, NSURLSessionDataDelegate {
         }
     }
 
-    // send request and build response object
+    // this methods can be removed in Swift 1.2
```

Comment:
In Swift 1.1, `func send(request:URLSession(=default):handler(=default):)` can't omit `URLSession` argument with trailing closure. On the other hand, we can omit it in Swift 1.2. Considering our opinions, following methods are suitable for each versions of Swift.

In Swift 1.1:
- `func send(request:handler(=default):)`
- `func send(request:URLSession:handler(=default):)`

In Swift 1.2:
- `func send(request:URLSession(=default):handler(=default):)`

I will leave a comment about this in code.